2.0/requirements.txt

```diff
@@ -1,4 +1,4 @@
 gunicorn==19.7.1
```

Comment:
The `CHANGELOG` for Gunicorn should be reviewed as part of this process to determine if it can be safely upgraded as part of the Django version bump.

See:

- http://docs.gunicorn.org/en/stable/news.html
- http://docs.gunicorn.org/en/stable/2017-news.html